R/model.R

```diff
@@ -9,7 +9,7 @@ i2p_gp_tune_model <- function(path) {
 
 # define required stan data
 i2p_data <- function(prev, ab, vacc, init_ab,
-                     prob_detectable, ut = 14,
+                     prob_detectable, ut = 14, ht = 0,
```

Comment:
I think it would be good to have more explicit/descriptive variable names than `ut` and `ht` here (and possibly in the Stan model although comment in the variable descriptions could take care of that to some degree).

Reply:
yup agree.